server/RenderContext.cs

```diff
@@ -626,9 +626,37 @@ public void Render(XGraphics graphics)
                     }
                 }
                 timers.Add(new Timer("droyne"));
-#endregion
+                #endregion
+
+                #region droyne
+                //------------------------------------------------------------
+                // Non-Hiver Worlds
+                //------------------------------------------------------------
+                if (styles.nonHiverWorlds.visible)
+                {
+                    solidBrush.Color = styles.nonHiverWorlds.textColor;
+                    foreach (World world in selector.Worlds)
+                    {
+                        if (world.HasCode("Nh"))
```

Comment:
Looks like T5 doesn't really say how to record NIL - other than (<name>), for example Regina has (Amindii)2. Looks like the above suggestion would work with some extra check of H1, H2, etc. which I guess should end up being HvH1, HvH2, etc. anyway.